OrbitCore/SymbolHelper.cpp

```diff
@@ -139,6 +146,13 @@ bool SymbolHelper::LoadSymbolsCollector(std::shared_ptr<Module> module) const {
 
 bool SymbolHelper::LoadSymbolsUsingSymbolsFile(
     std::shared_ptr<Module> module) const {
+  if (module->m_DebugSignature.empty()) {
```

Comment:
It's only one print so I'm not sure, but would it be worth to factor this in a function so that the string is not duplicated? And then have something like
```
if (!CheckModuleHasDebugSignature(module)) {
  return false;
}
```
?

Reply:
I made the error message  even a little longer, so yes I also think it makes sense to factor it out -> done